qa/rpc-tests/excessive.py

```diff
@@ -146,10 +146,12 @@ def run_test (self):
             except JSONRPCException: # could be spent all the txouts
               pass
           print ntxs, " transactions"
-          time.sleep(1)
           self.nodes[random.randint(0,3)].generate(1)
-          time.sleep(1)
+          time.sleep(10)
```

Comment:
not sure why i changed that, i think i was just testing something out.  i'll change it back